src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -783,4 +773,7 @@ public AnnotatedType[] getAnnotatedExceptionTypes() {
                 getGenericExceptionTypes(),
                 TypeAnnotation.TypeAnnotationTarget.THROWS);
     }
+
+    record ParameterData(Parameter[] parameters, boolean real) {}
```

Comment:
The `ParameterData.parameters` field can probably be marked as [`@Stable`] (since the `parameters` array is never modified):
```suggestion
    record ParameterData(@Stable Parameter[] parameters, boolean real) {}
```

Also the `real` record component should probably be renamed to `isReal` as @liach suggested in [GH‑9143 (comment)].

[GH‑9143 (comment)]: https://github.com/openjdk/jdk/pull/9143#discussion_r899802093
[`@Stable`]: https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/jdk/internal/vm/annotation/Stable.java

Reply:
Done!

Reply:
I don't think this stable is critical here, as we don't use the parameters array entries separately, but doesn't seem wrong to have it too.